worth2/main/views.py

```diff
@@ -3,14 +3,22 @@
 from django.core.urlresolvers import reverse
 from django.views.generic.base import TemplateView
 from django.views.generic.list import ListView
-from django.shortcuts import get_object_or_404, redirect
+from django.shortcuts import get_object_or_404, redirect, render
 
 from pagetree.generic.views import PageView
+import quizblock.models as qm
```

Comment:
My feeling here is that renaming imports is essentially making a new variable, so I'd use the same logic on naming the import as if I were making a new variable at the same scope. Specifically, I stick with the concept of "variable span" from Code Complete, which is roughly counting the number of lines of code between where a variable is introduced or declared and the last usage of the variable (or where it goes out of scope, ideally). The length and clarity of the variable name should be roughly proportional to its span. Eg, a one or two letter variable is fine in a very short inner loop or function that's only a couple of lines long. Anything that will be (potentially) used across a wider swath of code needs a longer and more "english" name.